### ISSUE TITLE
Toolkit page: Add anchors to Guides and External Resources sections 

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2802]
### Overview
We need to be able to go directly to the Guides and External Resources sections on the [Toolkit page](https://www.hackforla.org/toolkit) so that we can link to those sections from the header and footer menus. For this issue, we will add anchors to those sections.

### Action Items
- [x] See this example page that has anchors and examine the anchors with developer tools: https://www.hackforla.org/about
- [x] In [pages/toolkit.html](https://github.com/hackforla/website/blob/gh-pages/pages/toolkit.html),
  - [x] Add an anchor to the Guides section so that https://www.hackforla.org/toolkit#guides will jump to that section
  - [x] Add an anchor to the External Resources section so that https://www.hackforla.org/toolkit#external-resources will jump to that section
- [x] For styling,
  - [x] First, read the [Standardized Components wiki page](https://github.com/hackforla/website/wiki/Standardized-Components). Note: The guide uses buttons as an example but the concept for reusable, standardized components is applicable to anchors. 
  - [x] Then, add any page specific SCSS classes for the anchors to [_sass/components/_toolkit.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_toolkit.scss)
- [x] Using Docker,
  - [x] Check that each anchor link jumps to its corresponding section
  - [x] Check that the changes look good on mobile, tablet, and desktop views using developer tools
  - [x] Check that any changes don't break anything on the website
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit issue #2393 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2393 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- [Toolkit page](https://www.hackforla.org/toolkit)
- [pages/toolkit.html](https://github.com/hackforla/website/blob/gh-pages/pages/toolkit.html)
- [_sass/components/_toolkit.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_toolkit.scss)
- Example page that has anchors: https://www.hackforla.org/about
- [Standardized Components wiki page](https://github.com/hackforla/website/wiki/Standardized-Components)
- Tracked in https://github.com/hackforla/website/issues/2481#issuecomment-968349924
- Related to issues
  - https://github.com/hackforla/website/issues/2393
  - https://github.com/hackforla/website/issues/2455

**Comment by Carlos-A-P** [comment]
Availability: 1 week
ETA: 4/20/2022

**Comment by Sparky-code** [comment]
Hello @ExperimentsInHonesty, while reviewing the PR for this issue I noticed that the buttons for 'suggest a guide' and 'suggest a resource' on the toolkits page don't actually connect to anything and appear broken, though they just don't have links. 

I understand you're the stakeholder here so I wanted to get your input on whether it would make sense to have something like an onclick class for those buttons that give the user visual feedback like "coming soon!" etc. so that they know the buttons aren't broken. Otherwise if the forms/links for that are in the pipeline and it doesn't make sense to insert that extra issue. 

My thought process is that a class like this would be useful for pages/elements that are live but in progress as a reusable 'construction sign' interaction. 

Thank you!

**Comment by ExperimentsInHonesty** [comment]
~@Sparky-code did the issue that you raised [above](https://github.com/hackforla/website/issues/2802#issuecomment-1110506971) get made into an actual issue?  If yes, please put a link to it here~

Nevermind.  I found it https://github.com/hackforla/website/issues/3115

**Comment by JessicaLucindaCheng** [comment]
- In addition to #3115, issue #3678 was created to temporarily hide the Suggest a guide and Suggest a resource buttons
